utils/qemu/Makefile

```
@@ -9,10 +9,10 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=qemu
-PKG_VERSION:=8.1.2
+PKG_VERSION:=8.2.0
 PKG_RELEASE:=1
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
-PKG_HASH:=541526a764576eb494d2ff5ec46aeb253e62ea29035d1c23c0a8af4e6cd4f087
+PKG_HASH:=bf00d2fa12010df8b0ade93371def58e632cb32a6bfdc5f5a0ff8e6a1fb1bf32
 PKG_SOURCE_URL:=http://download.qemu.org/
```

```suggestion
PKG_SOURCE_URL:=https://download.qemu.org/
```
You could probably update this at the same time, it is easily checked by opening https://download.qemu.org/ in your browser